Release notes for Platform release 800xx (#2570)
harness/developer-hub: release-notes/platform.md

Harness deploys changes to Harness SaaS clusters on a progressive basis. This means that the features and fixes that these release notes describe may not be immediately available in your cluster. To identify the cluster that hosts your account, go to the **Account Overview** page. 
:::

## Latest - July 21, 2023, version 800xx

```mdx-code-block
<Tabs>
  <TabItem value="What's new">
```

> - Earlier, when an administrator enabled the account-level two-factor authentication (2FA) setting, users were affected in the following way:

Reviewer: ```suggestion
- Earlier, when an administrator enabled the account-level two-factor authentication (2FA) setting, it impacted the users in the following manner:
```

Author: "Impacted" is a good suggestion, but it's the stronger of the two words in terms of effect. "Affected" sounds a little less... what's the word... harmful or loud? I like the active construction, though, as long as what "it" refers to is abundantly clear.

  1. Users who had elected to use that account as their default account received 2FA emails, and the user-level 2FA setting was enabled in all their profiles. The users were not allowed to disable the setting.

>   2. Harness made the user-level setting available  only when an administrator disabled the account-level setting subsequently. Even then, the user-level 2FA setting remained enabled, and users continued to receive a 2FA challenge until they manually disabled the user-level setting in their profiles. (PL-39507, ZD-46268)

Reviewer: ```suggestion
  2. Harness allowed users to modify the 2FA setting only if an administrator disabled the account-level setting subsequently. Even then, the user-level 2FA setting remained enabled, and users continued to receive a 2FA challenge until they manually disabled the user-level setting in their profiles. (PL-39507, ZD-46268)
```

Author: Good suggestion, Charanya! I would retain "when", though; it's not a question of only if here but also when. With "when", the "if" is implicit but not the other way round. Does that make sense?

>   This behavior has been remediated. When an administrator enables the account-level 2FA setting, Harness sends the users 2FA emails but does not enable the user-level 2FA settings. Users are free to enable or disable the user-level setting in their profiles. When a user attempts to log in to their Harness account, Harness presents them with a 2FA challenge only if one or both of the settings (the account-level setting and the user-level setting) are enabled. If both settings are disabled, Harness does not present a 2FA challenge.

Reviewer: ```suggestion
This behavior has been remediated. When an administrator enables the account-level 2FA setting, users will receive 2FA emails from Harness.  However, the user-level 2FA setting is not enabled. Users are free to enable or disable the user-level setting in their profiles. When a user attempts to log in to their Harness account, Harness presents them with a 2FA challenge only if one or both of the settings (the account-level setting and the user-level setting) are enabled. If both settings are disabled, Harness does not present a 2FA challenge.
```

Author: Good suggestions, again. Let's drop the "will" because, as of the time users read this, the new behavior is in effect. Also, I tried to write in the active voice, which made it necessary for me to mention "Harness" as the actor. Is passive better there?

Harness deploys changes to Harness SaaS clusters on a progressive basis. This means that the features and fixes that these release notes describe may not be immediately available in your cluster. To identify the cluster that hosts your account, go to the **Account Overview** page. 
:::

## Latest - July 27, 2023, version 80022

```mdx-code-block
<Tabs>
  <TabItem value="What's new">
```
- Earlier, when an administrator enabled the account-level two-factor authentication (2FA) setting, it affected users in the following manner:
  1. Users who had set that account as their default account received 2FA emails, and the user-level 2FA setting was enabled across all their profiles. The users were not allowed to disable the setting.
  2. Harness allowed users to modify the 2FA setting only when an administrator disabled the account-level setting subsequently. Even then, the user-level 2FA setting remained enabled, and users continued to receive a 2FA challenge until they manually disabled the user-level setting in their profiles. (PL-39507, ZD-46268)

>   This behavior has been remediated. When an administrator enables the account-level 2FA setting, Harness sends the users 2FA emails but does not enable the user-level 2FA settings. Users are free to enable or disable the user-level setting in their profiles. When a user attempts to log in to their Harness account, Harness presents them with a 2FA challenge only if one or both of the settings (the account-level setting and the user-level setting) are enabled. If both settings are disabled, Harness does not present a 2FA challenge.

Reviewer 2: ```suggestion
  This behavior has been remediated. When an administrator enables the account-level 2FA setting, Harness sends the users 2FA emails but does not enable the user-level 2FA settings. Users can now enable or disable the user-level setting in their profiles. When a user attempts to sign in to their Harness account, Harness presents them with a 2FA challenge only if one or both of the settings (the account-level setting and the user-level setting) are enabled. If both settings are disabled, Harness does not present a 2FA challenge.
```
